scripts/jmwalletd.py

```diff
@@ -0,0 +1,419 @@
+#! /usr/bin/env python
+
+import datetime
+import os
+import time
+import abc
+import json
+import atexit
+from io import BytesIO
+from twisted.python.log import startLogging
+from twisted.internet import endpoints, reactor, ssl, task
+from twisted.web.server import Site
+from twisted.application.service import Service
+from klein import Klein
+
+from optparse import OptionParser
+from jmbase import get_log
+from jmbitcoin import human_readable_transaction
+from jmclient import Maker, jm_single, load_program_config, \
```

Comment:
missing `StoragePasswordError` from jmclient

Reply:
Thanks, done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/670/commits/5b0bf4ae2abeded214053f8bc54392cdae2897ba